config/config.exs

```diff
@@ -31,9 +31,24 @@ config :mdns_lite,
   ]
 
 if Mix.env() == :test do
-  # Disable the IP address monitor for mdns_lite unit tests (this is a no-op)
+  # Allow Responders to still be created, but skip starting gen_udp
+  # so tests can pass
   config :mdns_lite,
-    ip_address_monitor: {Agent, fn -> nil end}
+    skip_udp: true
+
+  # Overrides for unit tests:
+  #
+  # * udhcpc_handler: capture whatever happens with udhcpc
+  # * udhcpd_handler: capture whatever happens with udhcpd
+  # * resolvconf: don't update the real resolv.conf
+  # * persistence_dir: use the current directory
+  # * bin_ip: just fail if anything calls ip rather that run it
+  config :vintage_net,
+    udhcpc_handler: VintageNetTest.CapturingUdhcpcHandler,
+    udhcpd_handler: VintageNetTest.CapturingUdhcpdHandler,
```

Comment:
This line doubled up somehow. Same with the doc for it above.

Reply:
I just want to point a finger here and say this is 100% copy/paste from https://github.com/nerves-networking/vintage_net/blob/master/config/config.exs#L5-L17 😂 

(in good spirit - I can totally fix)

Reply:
Done